clang/tools/clang-offload-bundler/ClangOffloadBundler.cpp

```diff
@@ -996,7 +996,14 @@ static FileHandler *CreateFileHandler(MemoryBuffer &FirstInput) {
   if (FilesType == "s")
     return new TextFileHandler(/*Comment=*/"#");
   if (FilesType == "o" || FilesType == "oo")
+  {
+    // Since windows does not support incremental linking
+    // use BinaryFileHandler to bundle object files.
+    llvm::Triple T(getTriple(TargetNames[HostInputIndex]));
+    if (T.getOS() == Triple::Win32)
```

Comment:
Odd.., I added a minor comment here and it disappeared. Adding it again.
This line could be changed this way:
```suggestion
    if (T.isOSWindows())
```

Reply:
I don't have any other comments except that minor one.
This review will need Alexey Bader's, he has merge permissions and also may provide more feedback.